### ISSUE TITLE
Incorrect application of PDG rounding rules?

### ISSUE THREAD

**Issue by anntzer** [lebigot/uncertainties#36]
```
In [7]: format(ufloat(724.2, 26.4), "")
Out[7]: '724.2+/-26.4'
```
I'd expect 724+/-26 if my reading of the PDG rules is correct (and anyways it makes sense to avoid printing an uncertainty with 3 significant digits :-)).

**Comment by lebigot** [comment]
Thank you for the feedback. I expect the same output, from my understanding of the [Particle Data Group convention](http://pdg.lbl.gov/2010/reviews/rpp2010-rev-rpp-intro.pdf), so I will have a look at the code.

**Comment by lebigot** [comment]
I had a look: this is actually a _feature_ I decided to have, so as to be consistent with Python's own behavior of always adding at least one digit after the decimal point in some cases (`print 724.` print `724.0`). This was not specified in the documentation, so [I qualified the default behavior](https://github.com/lebigot/uncertainties/blob/master/doc/user_guide.rst#printing)—which is to "usually" print with the Particle Data Group rules, when the user does not specify anything. What do you think?

**Comment by anntzer** [comment]
I first checked that the PDG itself actually follows its rules in this case: see http://pdg.lbl.gov/2015/tables/rpp2015-sum-gauge-higgs-bosons.pdf note (l) on the last page (60+/-5).

I personally think that printing an error with three significant digits looks "more" wrong than a float lacking a decimal point; I don't really care whether you write `724+/-26` or `724.+/-26.`.

**Comment by lebigot** [comment]
I implemented the PDG rule for the case that opened this issue:
```
>>> print uncertainties.ufloat(724.2, 26.4)
724+/-26
```
However, I kept the default Python behavior of "at least one digit past the decimal point" when printing numbers with a zero uncertainty (`print ufloat(724, 0)` prints `724.0+/-0`).

This is implemented in revision 7f5dce5. This will be in version 2.4.7.2.
